packages/origin-ui-core/src/components/bundles/BudleContents.tsx

```diff
@@ -55,7 +57,6 @@ const cardSelectedColor = '#3a1c47';
 const cardColor = '#3e3e3e';
 const cardHeaderColor = '#3b3b3b';
 const cardHeaderSelectedColor = '#9b00c8';
```

Comment:
I know this is not a part of this PR, but could we also reuse the colors from `/src/styles/variables.scss` instead of defining these new ones? If we can't, then add these new colors to variables.scss instead of keeping them in the .tsx file please.